kickstarter/android-oss
# 🐛payment method UI bug

# What ❓
- Fixed bug where you can't see horizontal progress bars on older devices. This is a pretty gross fix but the only other solution is creating a custom `ProgressBar`, which also feels wrong. You can read more about it here https://stackoverflow.com/questions/14171471/remove-vertical-padding-from-horizontal-progressbar. The second solution did not work for older devices.
- Changed indeterminate horizontal progress bar style from `?android:attr/progressBarStyleHorizontal` to `Widget.AppCompat.ProgressBar.Horizontal`
- No longer allowing users to click the save button if the progress bar is visible when saving new payment method.
- Refactored logic to determine if a card is acceptable; instead of depending on the card form, depending on the card number.


# How to QA? 🤔
Add a new payment method and visually confirm you can see the progress bar and the save button is disabled when it's visible.

# See 👀
<img src="https://user-images.githubusercontent.com/1289295/52740644-77f07c00-2fa1-11e9-9bd2-67f92e4652cf.png" width="330"/>
